core/settings.py

```diff
@@ -197,7 +197,8 @@
 
 CACHES = {
     "default": {
-        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
+        "LOCATION": f"rediss://:{os.environ.get('REDIS_AUTH_TOKEN')}@{os.environ.get('REDIS_PRIMARY_ENDPOINT_ADDRESS')}/0",  # noqa: E501
```

Comment:
is the double s here a typo?

Also, what do you want to happen if REDIS_AUTH_TOKEN / REDIS_PRIMARY_ENDPOINT_ADDRESS are not set? At the moment it will construct an invalid connection string. Can we keep the LocMemCache as a fall back for local dev?

Reply:
`rediss://` indicates a secure connection with tls rather than a standard one. Yes, this is just an intial change to test the functionality within the dev environment. I will add logic to fall back to memcache if local development or variables are not present.